qa/tasks/salt_manager.py

```diff
@@ -27,6 +28,19 @@
 master_role = 'client.salt_master'
 
 
+def systemctl_status(self, service_name):
```

Comment:
Interesting this method can be probably reused in `__systemctl_remote`.

Reply:
Good idea. Done!